src/components/accounts/recovery_setup/SetupRecoveryMethod.js

```diff
@@ -128,6 +137,7 @@ class SetupRecoveryMethod extends Component {
             location,
         } = this.props;
 
+        Mixpanel.track("SR Setup recovery method start")
```

Comment:
Probably worth moving this inside of `this.state.success` and include tracking for `this.method`